docs/usage/fields.md

```diff
@@ -0,0 +1,85 @@
+
+The `Field` class is used to define fields in models.
+
+TODO: Refactor the `Field customization` section in the `json_schema.md`.
+
+## Alias fields
+
+For validation and serialization, you can define an alias for a field.
+
+There are three ways to define an alias:
+
+* `Field(..., alias='foo')`
+* `Field(..., validation_alias='foo')`
+* `Field(..., serialization_alias='foo')`
+
+The `alias` parameter is used for both _validation_ and _serialization_. The `validation_alias` and `serialization_alias`
+parameters are used for _validation_ and _serialization_ respectively.
+
```

Comment:
I think around this point I would add a little info callout talking about Intellisense in vscode and how it will use the value provided in `alias` for the `__init__` signature. Then I would say something to the effect of "if you set `populate_by_name=True` in the model config, and want to use the field name in the `__init__`, use `Field(alias=str(<alias value>))`.

I might also mention that at this time it is not possible for Intellisense to reflect the value of `validation_alias` in the `__init__`, but if desired you could get the modified init signature by replacing `Field(validation_alias='myValidationAlias')` with `Field(alias='myValidationAlias', serialization_alias='my_field_name')`.

Reply:
For purposes of "flow", I can see how it might make sense to put such a callout at the end of this file rather than here.

But I feel this will be a fairly frequent point of frustration for users, so if possible I'd like to include it near the top to make it more likely for people to stumble upon it.